os/arch/arm/src/imxrt/imxrt_mtd_flash.c

```diff
@@ -251,10 +251,8 @@ static void imxrt_sectorerase(struct imxrt_dev_s *priv, off_t sector)
  * Name:  imxrt_bulkerase
  ************************************************************************************/
 
-static inline int imxrt_bulkerase(struct imxrt_dev_s *priv, unsigned long arg)
+static inline int imxrt_bulkerase(struct imxrt_dev_s *priv, size_t startblock, size_t nsectors)
```

Comment:
Bulkerase means that format entire of flash, you don't have to add start & nsector or any args.

Reply:
You're right. Reverted.